src/dotnet/commands/dotnet-run/RunCommand.cs

```diff
@@ -197,7 +198,7 @@ private ICommand GetRunCommand()
                 globalProperties.Add("TargetFramework", Framework);
             }
 
-            Project project = new Project(Project, globalProperties, null);
+            var project = new ProjectInstance(Project, globalProperties, null);
```

Comment:
I'm curious, why did this need to change from Project to ProjectInstance?

Reply:
I was unable to prevent the globbing via the global properties in this manner when using a `Project` vs. a `ProjectInstsance`.  I haven't yet digged into whether or not this is a bug in the MSBuild API; I did check with @rainersigwald first that a `ProjectInstance` should probably be used here anyway.